### PR TITLE
ENT-6490: Update Node Docker instruction for OS 4.9

### PR DESCRIPTION
**Pull request by vkolomeyko**
Changing instructions for OS 4.9.
Once this is reviewed by @corda/technical-writers then this change should be reflected on versions of Corda OS from 4.6 onwards.

### CODE REVIEW

File: content/en/platform/corda/4.9/community/node-docker-deployments.md [hunk 1]
```diff
@@ -138,7 +138,8 @@ There are some optional variables which allow customisation of the generated con
 * `MY_P2P_PORT` - The port to advertise the node on (defaults to 10200). If changed, ensure the container is launched with the correct published ports.
 * `MY_RPC_PORT` - The port to open for RPC connections to the node (defaults to 10201). If changed, ensure the container is launched with the correct published ports.
 
-Once the container has finished performing the initial registration, the node can be started as normal
+Once the container has finished performing the initial registration, it should be possible to initialize node's DB
```
Comment (adelel1) [inline content/en/platform/corda/4.9/community/node-docker-deployments.md:141]: I would rephrase this, sounds like we're not sure if it will work, i.e. 'should'.
Are there any scenarios where this wont work?

**Comment by vkolomeyko** [comment]
Superseded by #625 - I am closing this PR without merging.
